contrib/upload-kktix-ticket-csv-to-bigquery.py

```diff
@@ -184,14 +341,15 @@ def sanitize_column_names(df: pd.DataFrame) -> pd.DataFrame:
 
     # pre-process of name uniqueness
     duplicate_column_names = find_reformat_none_unique(style_reformatted_columns)
-    logging.info(
-        f"Found the following duplicate column names: {duplicate_column_names}"
-    )
+    if duplicate_column_names:
+        logging.error(
+            f"Found the following duplicate column names: {duplicate_column_names}"
+        )
```

Comment:
```suggestion
        logging.error(
            "Found the following duplicate column names: %s",
            duplicate_column_names,
        )
```

See https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/

Reply:
Thanks for sharing this article! It's new and interesting concept for me. Lesson learned! I will batch update the f-string in a commit later.